pkg/networking/trafficplugin/protocol/tcp/tcp.go

```diff
@@ -81,7 +81,7 @@ func (p *TCP) Process() {
 	}
 	inv.Strategy = util.GetStrategyName(p.SvcNamespace, p.SvcName)
 	inv.Args = p.UpgradeReq
-
+	klog.Infof("MicroServiceName:%s, Endpoint:%s, Port:%s, SourceServiceID:%s, Protocol:%s, Strategy:%s, Args:%v", inv.MicroServiceName, inv.Endpoint, inv.Port, inv.SourceServiceID, inv.Protocol, inv.Strategy, inv.Args)
```

Comment:
Do we need to add any log level for it?

Reply:
Has been changed to modify klog.V(4).Infof, can it?

Reply:
@fisherxu /ptal